sidebars.ts

```diff
@@ -2,18 +2,27 @@ import type {SidebarsConfig} from '@docusaurus/plugin-content-docs';
 
 
 const sidebars: SidebarsConfig = {
-  troubleshootingSidebar: [{type: 'autogenerated', dirName: 'troubleshooting'}], // Adjust `dirName` as needed
-  changelogSidebar: [{type: 'autogenerated', dirName: 'changelog'}], // Adjust `dirName` as needed
-  docsSidebar : [
-    {
-      type: 'category',
-      label: 'Getting Started',
-      collapsed: false,
-      items: ['index', {type: 'category', label: "Before You Begin", items: [
-        'intro/before-you-begin/setting-up-flutterflow', 'intro/before-you-begin/app-architecture','intro/quickstart'
-    ]}],
-    }
-  ]
+    troubleshootingSidebar: [{type: 'autogenerated', dirName: 'troubleshooting'}], // Adjust `dirName` as needed
+    changelogSidebar: [{type: 'autogenerated', dirName: 'changelog'}], // Adjust `dirName` as needed
+    docsSidebar: [
+        {
+            type: 'category',
+            label: 'Getting Started',
+            collapsed: false,
+            items: ['index', {
+                type: 'category', label: "Before You Begin", items: [
+                    'intro/before-you-begin/setting-up-flutterflow', 'intro/before-you-begin/app-architecture', 'intro/quickstart'
+                ]
+            }, {
+                type: 'category', label: "Understanding The FlutterFlow UI", items: [
+                    {
+                        type: 'autogenerated',
+                        dirName: 'understanding-the-flutterflow-ui'
```

Comment:
I added and pushed these changes for you @johnbhiggins7 
Earlier we were adding page names in the index list, its not really required, we can autogenerated based on the folder name itself. This trick doesnt work for intro docs, but any other folder, it works!